assets/airports/kstl.json

```diff
@@ -0,0 +1,6728 @@
+                "12L": [],
+                "12R": [],
+                "24": [],
+                "29": [],
+                "30L": [],
+                "30R": []
+            },
+            "body": ["CSX"],
+            "exitPoints": {
+                "BDF": ["SKYPE", "SKUTR", "BDF"],
+                "LEBOY": ["DEECE", "LEBOY"],
+                "NEENS": ["MYKEY", "NEENS"],
+                "SPI": ["TEWHY", "SPI"]
+            },
+            "draw": [
+                ["CSX", "SKYPE"],
+                ["CSX", "DEECE"],
+                ["CSX", "MYKEY"],
+                ["CSX", "TEWHY"],
+                ["BDF*"],
+                ["LEBOY*"],
+                ["NEENS*"],
+                ["SPI*"]
+            ]
+        },
+        "CHUUC3": {
+            "icao": "CHUUC3",
+            "name": "Chuck Three",
+            "rwy": {
+                "6": [],
+                "11": [["LUUBE", "A25+"], "DAFUS", "GEJAY"],
+                "12L": [["HEFNR", "A25+"], "DAFUS", "GEJAY"],
+                "12R": [["HEFNR", "A25+"], "DAFUS", "GEJAY"],
+                "24": [],
+                "29": [["HAGEE", "A25+"], "WAARN", "ELDBB"],
+                "30L": [["PYUNG", "A25+"], "ALLNT", "ELDBB"],
+                "30R": [["PYUNG", "A25+"], "ALLNT", "ELDBB"]
+            },
+            "body": ["CHUUC"],
+            "exitPoints": {
+                "ARG": ["MYERZ", "ARG"],
+                "MAW": ["MYERZ", "MAW"],
+                "MYERZ": ["MYERZ"]
+            },
+            "draw": [
+                ["GEJAY", "CHUUC", "MYERZ*"],
+                ["WAARN", "ELDBB", "CHUUC"],
+                ["MYERZ", "ARG*"],
+                ["MYERZ", "MAW*"]
+            ]
+        },
+        "DRUSE3": {
+            "icao": "DRUSE3",
+            "name": "Droose Three",
+            "rwy": {
+                "6": [],
+                "11": [["LUUBE", "A25+"]],
+                "12L": [["HEFNR", "A25+"]],
+                "12R": [["HEFNR", "A25+"]],
+                "24": [],
+                "29": [["HAGEE", "A25+"], "WAARN", "ELDBB", "BERAA"],
+                "30L": [["PYUNG", "A25+"], "ALLNT", "ELDBB", "BERAA"],
+                "30R": [["PYUNG", "A25+"], "ALLNT", "ELDBB", "BERAA"]
+            },
+            "body": ["DRUSE"],
+            "exitPoints": {
+                "BNA": ["PLESS", "DENNI", "TINGS", "LAJUG", "BNA"],
+                "DENNI": ["PLESS", "DENNI"]
+            },
+            "draw": [
+                ["HEFNR", "DRUSE", "PLESS", "DENNI*"],
+                ["DENNI", "TINGS", "LAJUG", "BNA*"],
+                ["ELDBB", "BERAA", "DRUSE"],
+                ["LUUBE", "DRUSE"]
+            ]
+        },
+        "EEMOS3": {
+            "icao": "EEMOS3",
+            "name": "Emose Three",
+            "rwy": {
+                "6": [],
+                "11": [["FUNKE", "A25+"], "BEART", "GMSKI"],
+                "12L": [["FUNKE", "A25+"], "BEART", "GMSKI"],
+                "12R": [["FUNKE", "A25+"], "BEART", "GMSKI"],
```

Comment:
Particularly with this chart, I think we could do with a `_RWY30L01DME` since the runway heading climb is a little over 500 feet (barely enough to necessitate it, but here it could be noticeable. If you wanted to go crazy, my quick math says _0.75nm_ is actually about what we need to get it close to the chart. And then we can use it a bunch of other places where that initial climb along runway heading is used.